carAPI/src/main/java/com/everis/entity/CarDto.java

```diff
@@ -5,9 +5,9 @@
 public class CarDto {
 
 	private String id;
-	private String brand;
+	private Brand brand;
```

Comment:
Should be BrandDTO  and CountryDTO. Because of the same reason as we have CarDto